### DISCUSSION TITLE
Mining rights for reported IDs?

### DISCUSSION THREAD

**Discussion by rioda-org** [idena-network/idena-go#1144] [Polls]
Mining rights for reported IDs?

Identities with reported flips are allowed to mine, they only loose session rewards. Some pools (as well as some solo-miners) abuse that loophole and submit reportworthy flips with low effort that get consensus no matter that (like 1,2,3,4). Should those identities be suspended to loose mining rewards as well?

Copied from oracle
https://scan.idena.io/contract/0x0b26a240109e8a4c5bace502691a4919de660a99

As few members in discord agree, this could have good discussion

**Comment by NPC69** [comment]
The problem is that this one doesn't go into specifics. How would we differentiate between people who honestly tried to make their flips good but failed and those who deliberately made bad flips?

**Reply by rioda-org** [reply]
That's why I added another option, "Alow mining if only one flip is reported"
That makes sense to me, if just one is reported, then, it's should be an honest creator mistake, or concensus fluke.

**Reply by NPC69** [reply]
Makes sense, a simple solution.

**Reply by NPC69** [reply]
However, I'm not sure if that would completely prevent them from making shitflips or if it would merely slow them down. It would definitely be interesting to watch the effects. Btw, I once got two flips reported in a single session due to bad actors. :D

**Comment by morienzo** [comment]
Punishing shitflip creators through mining is a very good solution. I choose the option "Allow mining if only one flip is reported" because to err is human (and there is already a significant penalty for this).

**Comment by hlolve** [comment]
This was discussed in discord.

"Flips are the key element of Idena network security."
"The protocol imposes severe penalties on users that create flips in breach of the rules."

Proposal: Suspend identities that intentionally submit reportable flips

IDs with status "verified" or "human" will be suspended if intentionally submit reportable flips

If in validation:

3 flips submitted and 2 reported
4 flips submitted and 2 reported
5 flips submitted and 3 reported

The IDs will get status "suspended"

As example of this attitude:
Resend always same flips without considering words pair.
Pool 0xee2697e512c90Da18281fA7613Cb102efb72D810

**Reply by hlolve** [reply]
The other error in the oracle is that a "newbie" can´t be "suspended". Will need a new status, only suspended for mining so also disable to submit flips, but allowed to participate in next validation without be terminated by the network.

**Comment by QuickMythril** [comment]
i would like to add to this, that sometimes when i don't feel like making flips, i submit blank ones.  i do this because i am willing to sacrifice the session rewards, and still earn the mining rewards.  i have actually been anticipating and looking out for this change to be implemeted eventually.  i guarantee i would NOT be submitting generic empty flips if i could not mine.

however i see the potential for abuse here.  couldn't a set of users agree to report as many flips as possible, in order to earn more of the share for themselves?  i also have had lots of frustration when trying really hard to make good flips, and they still get reported for unknown reasons.  that would be double punishment of the flip creator, for someone else misusing the reports.

i have mixed feelings about this idea overall, but making good flips is basically the security of the chain, so it's probably for the best to do it.  maybe it will be the motivation people need to try harder (or just give up and go away if they have no intention to do it properly.)

**Reply by NPC69** [reply]
What exactly here do you think would incentivise them to purposefully report maliciously? I think such behaviour is already happening.

**Reply by hlolve** [reply]
We already have abuse of reports, random reports is a known case. A person can random report and maybe earn rewards without any punishment or cost. The only way to use correctly the report tool, is an educated user and interested in the healthy of the network.
